src/operators/infinitepepo.jl

```diff
@@ -111,10 +111,8 @@ TensorKit.space(T::InfinitePEPO, i, j) = space(T[i, j, end], 1)
 function initializePEPS(
     T::InfinitePEPO{<:PEPOTensor{S}}, vspace::S
 ) where {S<:ElementarySpace}
-    Pspaces = Array{S,2}(undef, size(T, 1), size(T, 2))
-    for i in axes(T, 1)
-        j in axes(T, 2)
-        Pspaces[i, j] = space(T, i, j)
+    Pspaces = map(Iterators.product(axes(T, 1), axes(T, 2))) do (r, c)
+        return space(T, r, c)
     end
```

Comment:
```suggestion
    Pspaces = map(I -> space(T, I.I...), eachindex(IndexCartesian(), T))
```

Reply:
Sorry, wasn't paying attention when I commited the suggestion, but I specifically only want to iterate over the first 2 or the three axes so I don't think this will work?